src/modules/map/components/infoButton/InfoButton.js

```diff
@@ -29,18 +29,25 @@ export class InfoButton extends BaElement {
 			{ name: translate('map_info_button_about'), url: 'https://geoportal.bayern.de/geoportalbayern/seiten/impressum.html' }
 		];
 
-		const openPopup = () => {
-
+		const openPopup = () => {			
 			// set created popup visible
 			var popup = this.shadowRoot.getElementById('info-popup');
-			if (popup.getAttribute('type') === 'hide') {
-				popup.setAttribute('type', 'show');
+			if (!popup.isOpen()) {
+				popup.openPopup();
 			}
 			else{
-				popup.setAttribute('type', 'hide');
+				popup.closePopup();
 			} 
 		};
 
+		// close popup on map click
+		window.addEventListener('map_clicked', () => {
+			var popup = this.shadowRoot.getElementById('info-popup');
+			if (popup.isOpen()) {
+				popup.closePopup();
+			}
+		}); 
+
```

Comment:
We should remove that, because it uses a CustomEvent, which is only fired for demonstration purposes.
A better approach in the future  will be using the state store, e.g. listening to changes of the map position.
Btw: we would also need a test for this section

Reply:
Okey didin't know that. So we can do that after we implement the state store listening to map events.